exchanges/okx/okx_websocket.go

```diff
@@ -862,22 +835,25 @@ func (ok *Okx) AppendWsOrderbookItems(entries [][4]string) ([]orderbook.Item, er
 // quantity with a semicolon (:) deliminating them. This will also work when
 // there are less than 25 entries (for whatever reason)
 // eg Bid:Ask:Bid:Ask:Ask:Ask
-func (ok *Okx) CalculateUpdateOrderbookChecksum(orderbookData *orderbook.Base) int32 {
+func (ok *Okx) CalculateUpdateOrderbookChecksum(orderbookData *orderbook.Base, checksumVal uint32) error {
```

Comment:
This is receiving zero value checksum values. Also:
```
[ERROR] | WEBSOCKET | 12/09/2022 10:04:00 | exchange okx websocket error - okx websocket connection: timeout waiting for response with signature: 02rMdBakWsRtMooA2TbSGlXLGXwmGtMo
[ERROR] | WEBSOCKET | 12/09/2022 10:04:00 | exchange okx websocket error - orderbook depth not found for Exchange okx CurrencyPair: BTC-USDT AssetType: spot
```

The waiting for response is slowing down processing.

Reply:
Just on this one and its cause, I'm not sure why there is a need to wait for the response for subscriptions. When looking at this: https://www.okx.com/docs/en/#websocket-api-subscribe there does not appear to be any requirement to send an identifier to process subscription requests, and instead OKX just returned messages stating what subscriptions have been made. 

So I'm thinking you can remove the `ok.Websocket.Conn.SendMessageReturnResponse(randomID, input)` for `WsChannelSubscription`, and just use `k.Websocket.Conn.SendMessage(input)`, process subscription messages in `wsHandleData` which will help with the above message Shazbert is receiving.

Reply:
Yea, Having sent the subscription and other messages with ### SendMessage() is a better way since I am not receiving any response message from SendMessageReturnResponse() method, even after adding timeout.
I have tried to handle the AuthLogin as such, handling the login success message on the wsHandleData

Reply:
**`[ERROR] | WEBSOCKET | 12/09/2022 10:04:00 | exchange okx WebSocket error - order book depth not found for Exchange okx CurrencyPair: BTC-USDT AssetType: spot`**
order book push data like _books5_ are entering the **WsProcessUpdateOrderbook()** having an order book checksum of 0.
So, I think, that's why  it's raising such error

Reply:
I have made slight changes regarding those errors mentioned above .. take a look and share your review comment.